react-basics/react-basics_hello-world/src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./styles.css";
+
+export default function App() {
+  //return <h1>Hello World!</h1>;
+  return <h1>HelloWorld</h1>
```

Comment:
this should be replace with the new function now
`  return <HelloWorld></HelloWorld>;
`